### PR TITLE
62/docs/update readme

### LINKED ISSUE #62: Update README
## Why

**Update image which is used to confirm nvidia-docker installation**

Currently `nvidia/cuda:11.0-base` is used as example to confirm nvidia-docker installation (https://github.com/cvpaperchallenge/Ascender/blob/master/README.md#optional-nvidia-container-toolkit). However, this image is out of date instead `nvidia/cuda:11.0.3-base` should be used.

**Update FAQ** 

Sometime Docker container unintentionally uses cache and performs messy. In that case, build image again and recreate container is a nice way to solve; `sudo docker compose up --build --force-recreate`. As discussed in #60, add this info to FAQ. 

## Definition of Done

`README.md` is updated and the points written in "why" section are all covered.

## How

Update `README.md`.

### PR DESCRIPTION
**Pull request by YoshikiKubotani**
## Issue URL

#62

## Change overview

 - Update the nvidia docker image name 
 - Add a new description about the build cache problem to the FAQ section

## How to test

Just see the README.md and confirm the changes above has been made

## Note for reviewers

close both #62 and #60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ $ sudo apt install -y nvidia-docker2
 $ sudo systemctl restart docker
 ```
 
-If `sudo docker run --rm --gpus all nvidia/cuda:11.0-base nvidia-smi` works, installation succeeded.
+If `sudo docker run --rm --gpus all nvidia/cuda:11.3-base nvidia-smi` works, installation succeeded.
```
Comment (YoshikiKubotani) [inline README.md]: here need to change from `11.3` to `11.0.3`

File: README.md [hunk 2]
```diff
@@ -209,3 +209,7 @@ We plan to incorporate Poetry 1.2.0 into Ascender immediately after its release.
 ### Change the Python version to run CI jobs
 
 By default, CI job (GitHub Actions workflow) of Ascender is run against Python 3.8 and 3.9. If you want to change the target Python version, please modify [the matrix part of `.github/workflows/lint-and-test.yaml`](https://github.com/cvpaperchallenge/Ascender/blob/master/.github/workflows/lint-and-test.yaml#L18).
+
+### The change to Dockerfile has not been reflected correctly on the image build
```
Comment (gatheluck) [inline README.md]:
```suggestion
### When changes to the Dockerfile are not reflected
```

File: README.md [hunk 3]
```diff
@@ -209,3 +209,7 @@ We plan to incorporate Poetry 1.2.0 into Ascender immediately after its release.
...
+
+When you run `sudo docker compose up --build` after adding some change on the Dockerfile, you may find no changes has been made on the image built.  This may be because the `docker build` command has used the build cache.  When that happens, try using `sudo docker compose up --build --force-recreate`, instead.
```
Comment (gatheluck) [inline README.md]: @YoshikiKubotani 
When changes to the Dockerfile are not reflected, potentially there are two possibilities. 
1. Docker using cache at image build step.
2. Docker using already created container.

I think `sudo docker compose up --build --force-recreate`  will solve problem caused by 2, but might not solve caused by 1. So maybe write two command like bellow might be more more friendly: 

- `sudo docker compose build --no-cache`  then `sudo docker compose up` (solution for 1.)
- `sudo docker compose up --force-recreate` (solution for 2.)

Or run `sudo docker compose build --no-cache` then `sudo docker compose up --force-recreate` will solve both case.

**Review by gatheluck — APPROVED** [review]
Thank you! LGTM!

**Pull request merged**